### PR TITLE
Sql reviews

### PR DESCRIPTION
**Pull request by tinevancorenland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# becode_database_api
\ No newline at end of file
+# becode_database_api
+
+CREATE DATABASE IF NOT EXISTS DB_notes;
+
+CREATE TABLE IF NOT EXISTS DB_notes.notes (
+  noteID int PRIMARY KEY AUTO_INCREMENT,
```
Comment (krisderycke) [inline README.md]: zou 'int' en 'varchar' niet in hoofdletters moeten?

**Review by Lisonallie — COMMENTED** [review]
Hi Tine,
I also use 'text' for characters over 255. I read it somewhere in the first exercise with the database. Your code looks great!
I just have a question as to the reason you named everything with lower case characters? Also is the syntax for title correct? is it UNIQUE NOT NULL or NOT NULL UNIQUE or does it not matter? Why use IDENTITY instead of AUTO_INCREMENT?
It might help you in the next changes to indent your code so it's easier for you to read especially when we start putting more variables in. Great job!

File: README.md (hunk 1 above)
Comment (Lisonallie) [reply README.md]: > zou 'int' en 'varchar' niet in hoofdletters moeten?

not necessary

**Review by Lisonallie — APPROVED** [review]

**Comment by iliastic** [comment]
Why just 500 characters in the text and not more?

**Comment by iliastic** [comment]
Is there a reason why you wrote "noteID" ID in Capital letters?

**Comment by tinevancorenland** [comment]
> Hi Tine,
> I also use 'text' for characters over 255. I read it somewhere in the first exercise with the database. Your code looks great!
> I just have a question as to the reason you named everything with lower case characters? 
--> i like it short and simple and always start with lower case... just a habit :-)

Also is the syntax for title correct? is it UNIQUE NOT NULL or NOT NULL UNIQUE or does it not matter? 
--> i can't seem to find anything on the order being important. So I assume - based on nothing now - that it does not matter, but did you find that it does?? 

Why use IDENTITY instead of AUTO_INCREMENT?
--> advice I got from Raf! In his code, the auto-increment did not light up and he found information about it being outdated, and people using the identity description from now on...

> It might help you in the next changes to indent your code so it's easier for you to read especially when we start putting more variables in. Great job!
--> thanks, very correct, indentations coming!

Thanks for being so detailled here x

**Comment by KennethVDS** [comment]
Goe gedaan Tineeeee

**Comment by tinevancorenland** [comment]
> Is there a reason why you wrote "noteID" ID in Capital letters?

No, changed them to small and with an underscore!

**Comment by tinevancorenland** [comment]
> Goe gedaan Tineeeee

haha merci Kenneeeeeeeeth

**Comment by tinevancorenland** [comment]
> Why just 500 characters in the text and not more?

Upt it!

**Pull request merged**